project/Dependencies.scala

```diff
@@ -3,7 +3,7 @@ import sbt._, Keys._
 object Dependencies {
 
   val ScalaVersions = Seq("2.11.11", "2.12.2")
-  val AkkaVersion = "2.4.17"
+  val AkkaVersion = "2.4.18"
```

Comment:
In fact we do not know they do work with 2.5.1 as there are a few cases we are stricter with in graph stages now.